react-native/ios/RealmReact/RealmReact.mm

```diff
@@ -153,6 +153,7 @@ - (void)setBridge:(RCTBridge *)bridge {
 
           auto &rt = *static_cast<facebook::jsi::Runtime *>(bridge.runtime);
           auto exports = jsi::Object(rt);
+          rt.global().setProperty(rt, "__RealmFuncs", exports);
```

Comment:
This file is probably not being invoked anymore, or?  I assume the `react-native` directory will go away soon.

Reply:
Yeah. I moved it and actually that line was not needed anyway 🙈